prow/autobump.sh

```diff
@@ -1,93 +0,0 @@
-#!/bin/bash
-# Copyright 2018 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# bump.sh will
-# * ensure there are no pending changes
-# * optionally activate GOOGLE_APPLICATION_CREDENTIALS and configure-docker if set
-# * run //prow:release-push to build and push prow images
-# * update all the cluster/*.yaml files to use the new image tags
-
-set -o errexit
-set -o nounset
-set -o pipefail
-
-# TODO(fejta): rewrite this in a better language REAL SOON
```

Comment:
Thoughts on making this run the new tool (maybe move to / or /hack)? These scripts are often easier to find than bazel targets

Reply:
Added a hack/autobump.sh that just invokes autobumper.